pkg/openstack/routes.go

```diff
@@ -421,6 +421,9 @@ func (r *Routes) DeleteRoute(ctx context.Context, clusterName string, route *clo
 	} else {
 		// atomic route update
 		blackhole := route.Blackhole
+		if blackhole {
+			addr = string(route.TargetNode)
+		}
```

Comment:
Trying to get my read round this, but at first glance it doesn't look right to me.

When we add a route NextHop is an address. Why would it be a node name here, and why would it be different for blackhole routes?

Reply:
`route.TargetNode` stands to the node IP, when a node  name cannot be found:

https://github.com/kubernetes/cloud-provider-openstack/blob/5a9459935aa33136eb697a50f1c84502ad18a14b/pkg/openstack/routes.go#L86-L90

See also the logic, when there are no atomic routes:

https://github.com/kubernetes/cloud-provider-openstack/blob/5a9459935aa33136eb697a50f1c84502ad18a14b/pkg/openstack/routes.go#L396-L399

Reply:
Ok, I see. As that context is in a comment 300 lines away it could probably do with another comment here:

```golang
// TargetNode was set to the NextHop of the route instead of the node name in ListRoutes() if the node was not found
```